Revision en docs (#1323)
mercadopago/devsite-docs: guides/in-person-payments/mp-point/how-to-integrate.en.md

# How to integrate Mercado Pago Point 

In order to be able to process payments in the *integrated mode* with our Point devices, it is necessary to download the Mercado Pago application for Android and IOS.

There are two different possible situations to integrate Point: 

> 1) When you can access your application from the same device (smartphone or tablet) where the Mercado Pago application is installed. These integrations can be done vía Deep linking or Intent-based. 

Reviewer: ```suggestion
1) When you are able to access your application from the same device (smartphone or tablet) where the Mercado Pago application is installed. These integrations can be done via Deep linking or Intent-based. 
```

>
> * You must have the Mercado Pago application (From version 2.34 for Android and 2.32 for iOS).
> * Have a Point device.
> * The user must be logged in with their Mercado Pago account in the application. 
> * Avaliable for Android version 2.8.0 or superior, iOS version 1.7.0 or superior and only when it´s in a iOS 9 environment or superior.

## Integration via Deep Linking

> One of the ways to integrate with Mercado Pago is via Deep Linking. When said _link_ is called, it is intercepted as a _Point-handled address_ by the Mercado Pago application.

Reviewer: @romisoto - Romi, achei esse trecho um pouco confuso: _When said link is called_  esse "Said" está correto mesmo?

Author: neste caso "Said" é como dizer "o link que foi mencionado antes", é um pouco formal, mas está gramaticalmente correto

> In the "call" to this _link_ you can send different parameters that will be taken by the Mercado Pago application and will impact in the payment. Once you´ve made that call to the previously defined link, the user will be redirected to the Mercado Pago appllication to swipe the card and make the purchase. 

Reviewer: ```suggestion
In the "call" to this _link_ you can send different parameters that will be taken by the Mercado Pago application and will impact the payment. Once you´ve made that call to the previously defined link, the user will be redirected to the Mercado Pago application to swipe the card and make the purchase. 
```

} ’
```

Next, it is necessary to configure the `device_name` from the Mercado Pago application. It is used to identify your cell phone or tablet and relate it to your Mercado Pago account. In this way, you will know which device to send the payment order to.

> The next step consist in generating a payment order and sending it via API to the corresponding device. The user will see the order sent to the application in the screen of the device. This means that the user will be able to swipe the card at that moment and continue with the proccess. 

Reviewer: ```suggestion
The next step consist in generating a payment order and sending it via API to the corresponding device. The user will see the order sent to the application in the device's screen. It means that the user will be able to swipe the card at that moment and continue with the process. 
```

The parameters that can be included are: 

> * `amount`: The amount that will be charged to the client (\*).

Reviewer: ```suggestion
* `amount`: The amount that will be charged from the client (\*).
```

* `description`: Description of the operation (Máx.: 20 caracters) (\*).
* `device_name`: Name of the device in which you want to process the payment. (\*).
* `cc_type`: Card type. Credit or debit card (\*).

> * `external_reference`: The `external_reference` is reference code that will allow you to track the Mercado Pago `payment_id` in your system. 

Reviewer: ```suggestion
* `external_reference`: The `external_reference` is a reference code that allows you to track the Mercado Pago `payment_id` in your system. 
```

> * `disable_back_button`: True o False. This is used to define wether you want the order to close or not by clicking on the back button. 

Reviewer: ```suggestion
* `disable_back_button`: True or False. This is used to define whether you want the order to close or not by clicking on the back button. 
```

## Payment notifications

It is necessary that you send the `notification_url`, where you'll receive notifications about new payments and status updates generated.

> In the article [notifications](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/notifications/webhooks) you could find more information about it.

Reviewer: ```suggestion
For mor information, check the [notifications](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/notifications/webhooks) article.
```

## Point payments

> Point payments can be searched in the Payments API. You can find more information in the following article: [API's](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/payments/_payments_id/get)

Reviewer: ```suggestion
Point payments can be searched in the Payments API. For more informatio, check the [API's](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/payments/_payments_id/get) article.
```
